pkg/tests/tasks/security/certmanager/main_test.go

```diff
@@ -72,8 +73,10 @@ func setupCertManagerOperator(t test.TestHelper) {
 
 	t.LogStep("Install cert-manager-operator")
 	oc.ApplyString(t, certManagerOperatorNs, certManagerOperator)
-	oc.WaitPodReady(t, pod.MatchingSelector("name=cert-manager-operator", certManagerOperatorNs))
-	oc.WaitPodReady(t, pod.MatchingSelector("app=cert-manager", certManagerNs))
+	retry.UntilSuccess(t, func(t test.TestHelper) {
+		oc.WaitPodReady(t, pod.MatchingSelector("name=cert-manager-operator", certManagerOperatorNs))
+		oc.WaitPodReady(t, pod.MatchingSelector("app=cert-manager", certManagerNs))
+	})
```

Comment:
This isn't a good solution. With the retry, the test will now wait for 1 hour before failing. That's too much. You should introduce `WaitPodReadyWithOptions` and pass in a custom retry  count (3 minutes?) instead of adding a `retry.UntilSuccess` around `WaitPodReady`.

Reply:
BTW, if possible, we should wait for the Deployment to be Available instead of waiting for Pods. Actually, since this Deployment is being created via a Subscription/CSV, we should be waiting for one of their conditions instead of waiting for the Deployment, but I don't know if the Subscription/CSV exposes such a condition.

Reply:
OK, I'll take a look and find a better solution